pavelib/utils/test/utils.py

```diff
@@ -131,7 +132,10 @@ def check_firefox_version():
 
     # Firefox will be run as a local process
     expected_firefox_ver = "Mozilla Firefox " + str(MINIMUM_FIREFOX_VERSION)
-    firefox_ver_string = subprocess.check_output("firefox --version", shell=True).strip().decode('utf-8')
+    if six.PY2:
+        firefox_ver_string = subprocess.check_output("firefox --version", shell=True).strip().decode('utf-8')
```

Comment:
This version works in both python 2 and python 3.  Did you try it out before you made this change?